files/build_templates/swss.service.j2

```diff
@@ -1,18 +1,40 @@
 [Unit]
-Description=switch state service container
+Description=switch state service
 Requires=database.service
 After=database.service
 
 [Service]
-User={{ sonicadmin_user }}
 # Wait for redis server start before database clean
 ExecStartPre=/bin/bash -c "while true; do if [ \"$(/usr/bin/docker exec database redis-cli ping)\" == \"PONG\" ]; then break; fi; sleep 1; done"
 ExecStartPre=/usr/bin/docker exec database redis-cli -n 0 FLUSHDB
 ExecStartPre=/usr/bin/docker exec database redis-cli -n 1 FLUSHDB
 ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
-ExecStart=/usr/bin/{{docker_container_name}}.sh start
+
+{% if sonic_hwsku == 'ACS-MSN2700' %}
+ExecStartPre=/etc/init.d/sxdkernel start
+ExecStartPre=/usr/bin/mst start
+ExecStartPre=/etc/mlnx/msn2700 start
+{% elif sonic_hwsku == 'AS7512' %}
+ExecStartPre=-/etc/init.d/xpnet.sh stop
+ExecStartPre=/etc/init.d/xpnet.sh start
+{% endif %}
+
+# systemd allows only one parent process within service,
+# so we spawn both dockers from single bash parent
+ExecStart=/bin/bash -c "/usr/bin/{{docker_container_name}}.sh start & /usr/bin/syncd.sh start"
```

Comment:
what if we do ExecStartPre=/usr/bin/syncd.sh start?

what's the difference between your current solution?